ptx/sec_antider.ptx

```diff
@@ -124,7 +124,17 @@
       <!-- Todo: Fix this weird image -->
 
       <image xml:id="img_anti1" width="100%">
-        <description></description>
+        <shortdescription>A labelled formula for the indefinite integral/</shortdescription>
```

Comment:
Although I'd prefer it to be otherwise, the book uses American spelling, which means we should use 'labeled' instead of 'labelled'. (It looks wrong to me, too, but until I make a Canadian edition, we're stuck with it.)